tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -91,4 +91,23 @@ select to_int(8);
 select "new_planner";
 select *; -- {ErrorCode 1065}
 
+-- limit
+select '=== Test limit ===';
+select number from numbers(100) order by number asc limit 10;
+select '==================';
+select number from numbers(100) order by number*2 asc limit 10;
+select '=== Test limit n, m ===';
+select number from numbers(100) order by number asc limit 10, 10;
+select '==================';
+select number from numbers(100) order by number-2 asc limit 10, 10;
+select '=== Test limit with offset ===';
+select number from numbers(100) order by number asc limit 10 offset 10;
+select '==============================';
+select number from numbers(100) order by number/2 asc limit 10 offset 10;
+select '=== Test offset ===';
+select number from numbers(10) order by number asc offset 5;
+select '===================';
+select number from numbers(10) order by number+number asc offset 5;
+select number from numbers(10000000000000) limit 1;
```

Comment:
We did not push down the limit to data source in v2 planner. So let's remove this heavy test.

Reply:
got it